### PR TITLE
Added vignette on basic use and plotting

### PR DESCRIPTION
**Pull request by davidahirshberg**
Breaking changes to very obscure features of synthdid_plot. Also made slight changes to its default alpha parameters, as parts of the plot were hard to see clearly.

### CODE REVIEW

**Review by davidahirshberg — COMMENTED** [review]
pkgdown issues aside, it looks good to me

**Comment by davidahirshberg** [comment]
I fixed the warnings that needed fixing.  I'm good to merge if the pkgdown stuff is settled on azure.

File: vignettes/more-plotting.Rmd [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  plot(estimate, spaghetti.units=rownames(top.controls))
+```
+
+# Complex Plots
+We can even overlay multiple estimates in one plot and include a magnified view of some regions.
+We create a flipbook in which we bring synthetic diff-in-diff's control
+closer to California's trajectory to we can see how parallel it is clearly, overlay it completely
+so we can better compare it to the ADH synthetic control method's synthetic control,
+plot the ADH synthetic control estimate, and zoom in on a region around treatment onset
+in which we can see the ADH synthetic control possibly begin to diverge from California's trajectory.
```
Comment (erikcs) [inline vignettes/more-plotting.Rmd:55]: the ADH abbreviation isn't defined anywhere on this page

File: vignettes/synthdid.Rmd [hunk 2]
```diff
@@ -7,47 +7,132 @@ vignette: >
...
 
 ```{r setup}
 library(synthdid)
 library(ggplot2)
-library(mvtnorm)
 ```
 
-# Example usage
+# Example use 
+We estimate the effect of California Proposition 99 on cigarette consumption
```
Comment (erikcs) [inline vignettes/synthdid.Rmd:24]: This vignette gives an good intro on using the main functions, but I think there is much more room here for you to explain parts of the paper too... Like, you write one sentence " plot the 10 controls with the largest weights 𝜔𝑖", I don't know if the typical user remembers the greek letters from the paper.

It does not hurt to have a recap here in the intro - you can add as much latex math as you want (within reason of course...)

**Review by erikcs — COMMENTED** [review]
Sorry for this ex-post review, I read through the rendered pages after the merge

**Pull request merged**